### PR TITLE
Second tutorial: Connect the PIM

### PR DESCRIPTION
**Pull request by NolwennP**
Still missing some screenshots

### CODE REVIEW

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+So, we definitely recommend you to take a look at it and try to find a connector that can suit your use case. It will save you a lot of time!
+
+1. Navigate to the [marketplace](https://marketplace.akeneo.com/extensions/add-ons?edition=all&version=all&api_use=1&sort=date) website.  
+The results are already filtered to show you the API-based connectors only.
+1. In this list of connectors, try to find the one that can fit your use case best.
+
+You found what you were looking for? Great! You can now install your connector and go directly to the [third step of this tutorial](/getting-started/connect-the-pim-4x/step-3.html).
+
+Couldn't find any connector that suits you?  
+It is the case for our example in this tutorial. We want to connect the PIM to our ERP but the marketplace doesn't provide any connector that covers this use case.
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
It is the case of our example in this tutorial. We want to connect the PIM to our ERP but the marketplace doesn't provide any connector that covers this use case.  
```
only a suggestion here

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+You found what you were looking for? Great! You can now install your connector and go directly to the [third step of this tutorial](/getting-started/connect-the-pim-4x/step-3.html).
+
+Couldn't find any connector that suits you?  
+It is the case for our example in this tutorial. We want to connect the PIM to our ERP but the marketplace doesn't provide any connector that covers this use case.  
+Alright! So let's build it ourselves!
+
+## Build your very own connector
+
+In the case you can't find an already cooked connector on our marketplace, you will have to develop it.
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
In the event that you can't find a connector already cooked on our marketplace, you will have to develop it. 
```

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+In the case you can't find an already cooked connector on our marketplace, you will have to develop it. 
+
+Below are the macro steps to successfully go through this development.
+1. Have your connection use case in mind
+2. Find out what are the third-party connection possibilities
+3. Choose your integration frequency
+4. Explore our API reference
+5. Think about your architecture
+6. Ochestrate your global connector
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
6. Orchestrate your global connector
```

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+Those steps are detailed below, illustrated by our tutotrial example.
+
+### 1- Have your connection use case in mind
+
+We know it can seem a bit obvious but it really is the first step for a successful connection between the PIM and your third-party. You need to have at least the global scenario of your connection in mind.
+
+#### In this tutorial
+_You want to connect your ERP. Each day, new products are created into your ERP. You want those new products to be sent over to the PIM so that you can enrich them, in the PIM, the day after._
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
_You want to connect your ERP. Each day, new products are created in your ERP. You want those new products to be sent over to the PIM so that you can enrich them, in the PIM, the day after._
```

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+### 2- Find out what are the third-party connection possibilities
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
### 2- Find out what the third-party connection possibilities are
```

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+Take a closer look at the software you want to connect to. How can you extract the data you will need? Does it have an API? Does it allow exporting flat files?
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
Take a closer look at the software you want to connect the PIM to. How can you extract the data you will need? Does the software has its own web API? Does it allow to export flat files?  
```

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+Take a closer look at the software you want to connect to. How can you extract the data you will need? Does it have an API? Does it allow exporting flat files?  
+
+#### In this tutorial
+_In our example, let's say that you took a look at your ERP and discovered that, like many ERPs, it is a bit old-school. It only provides exports of products in XML files._
+
+### 3- Choose your integration frequency
+Define at which frequency you want your connector to integrate data into the PIM or retrieve it, if it is a destination. You need to choose this frequency carefully as it can have an impact on the PIM load.
+
+#### In this tutorial
+_In our example, as stated above, we know that every day, new products are created into the ERP. So we define that our connector will integrate new products, once a day at midnight, outside working hour, to lower as much as possible the PIM load._
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
_In our example, as stated above, we know that every day, new products are created in the ERP. So we define that our connector will integrate new products, once a day at midnight, outside working hours, to lower as much as possible the PIM load._
```

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+
+### 4- Explore our API reference
+In this step, you will need to find in our [API reference](api-reference-index.html) the endpoints you will need to achieve such a connection.
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
In this step, you will need to find in our [API reference](api-reference-index.html) the endpoints you will need to achieve such connection. 
```

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+_In our example, as stated above, we know that every day, new products are created into the ERP. So we define that our connector will integrate new products, once a day at midnight, outside working hour, to lower as much as possible the PIM load._
+
+### 4- Explore our API reference
+In this step, you will need to find in our [API reference](api-reference-index.html) the endpoints you will need to achieve such a connection. 
+
+#### In this tutorial
+_For our example, you will need to create and update products in the PIM so we will mainly need this endpoint: [Patch](/api-reference.html#patch_products) on products._
+
+### 5- Think about your architecture
+Choose the architecture you want for your connector. Indeed, you will have two options here:
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
Choose the architecture you want for your connector as you will have two options here:
```

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 10]
```diff
@@ -0,0 +1,120 @@
...
+    - You have access to the third-party code source and can develop your connector directly in it.
+    - You can otherwise create a middleware that will sit right in the middle of the tird-party you want to connect and the PIM.
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
    - You can otherwise create a middleware that will sit right in the middle of the third-party you want to connect, and the PIM.
```

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 11]
```diff
@@ -0,0 +1,120 @@
...
+
+#### In this tutorial
+_For our example, we choose the second option there is little chance that your ERP allows developing into its base code. So you will have to develop your connector as a middleware._
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
_For our example, we choose the second option as little chances are that your ERP accepts external development into its base code. Here, you will have to develop your connector as a middleware._
```
Just a suggestion, but you miss a word after "option" anyway.

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 12]
```diff
@@ -0,0 +1,120 @@
...
+
+### 6- Ochestrate your global connector
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
### 6- Orchestrate your global connector
```

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 13]
```diff
@@ -0,0 +1,120 @@
...
+
+A crucial step. In most cases, you will need to manually map some PIM information with the data sent or received in the software you want to connect.
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
A crucial step. In most cases, you will need to manually map some PIM information with the data sent/received in/from the software you want to connect.
```

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 14]
```diff
@@ -0,0 +1,120 @@
...
+
+#### In this tutorial
+_For our example, here is the global orchestration of our connector. First, every evening, you will extract the list of products that were created/updated during the day, in an XML file. Then, this file is read by your connector. ERP products are transformed into PIM products, by applying a mapping of the ERP information with the right attributes of the PIM. To finish, the connector sends the PIM products by using the [patch](/api-reference.html#patch_products) endpoint of our API._
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
_For our example, here is the global orchestration of our connector. First, every evening, you will extract the list of products that were created/updated during the day, in a XML file. Then, your connector reads this file. ERP products are then transformed into PIM products, by mapping the ERP information with the right PIM attributes. Lastly, the connector sends the PIM products by using the [patch](/api-reference.html#patch_products) endpoint of our API._
```

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 15]
```diff
@@ -0,0 +1,120 @@
...
+
+## Some pro tips
+
+As we want you to build the best connector ever, here are some tips to help you cook it the right way.
+
+### Start small
+The use case we took here is really simple, as you can see. In real life, connections maybe a bit more tricky with a lot of different use cases to handle.
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
The use case we took here is really simple, as you can see. In real life, connections may be a bit more tricky with a lot of different use cases to handle.
```

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 16]
```diff
@@ -0,0 +1,120 @@
...
+Another option is to provide a configuration file to input this configuration. It's usually faster to develop but keep in mind that you will be far from delivering the best UX for your final users. :wink:
+
+### Think scalability and performance
+
+It is really important that you take into consideration the scalability and performance of your connector.
+
+As stated previously, again, we advise you to start small. Benchmark your connector with a small volume, then test it with a real-life catalog volume.
+
+To help you in this task, here are examples of questions you can ask yourself:
+- _"How many products, assets, reference entities or options your connector will handle?"_
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]: I would link each entity to its definition in the documentation, as your readers may not know what reference entities are.

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 17]
```diff
@@ -0,0 +1,120 @@
...
+- _"How big are your products, assets or reference entities in termes of number of attributes completed?"_
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
- _"How big are your products, assets or reference entities in terms of number of attributes completed?"_
```

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 18]
```diff
@@ -0,0 +1,120 @@
...
+
+Also, don't forget to think about the fact that every API call you are doing has an impact on the **PIM load**. Always keep in mind that your connector won't be the only one connected to the PIM. Try to have a load print as low as possible. To do so, you can use caches for example.
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]: load footprint?

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 19]
```diff
@@ -0,0 +1,120 @@
...
+
+### Test, test and test
+
+Testing should not be optional. You will need to test your full use case and also focus on all the edge cases that can happen.
+
+### Use the PHP client
+
+If you want to use PHP to build your connector, we highly recommend the use of our PHP client.
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]: A link to our client would be awesome here! thanks.

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 20]
```diff
@@ -0,0 +1,120 @@
...
+
+It encapsulates a lot of complexity, like dealing with pagination or the authentication. In the end, you will save a lot of time by using it.
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
It encapsulates a lot of complexity, like dealing with pagination or authentication. In the end, you will save a lot of time by using it.
```

File: content/getting-started/connect-the-pim-4x/step-3.md [hunk 21]
```diff
@@ -0,0 +1,28 @@
+# _Step 3_ | Identify your connector in the PIM
+
+In the very first step of this tutorial, you created in the PIM a connection.
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-3.md]:
```suggestion
In the very first step of this tutorial, you created a connection in the PIM.  
```

File: content/getting-started/connect-the-pim-4x/step-3.md [hunk 22]
```diff
@@ -0,0 +1,28 @@
+# _Step 3_ | Identify your connector in the PIM
+
+In the very first step of this tutorial, you created in the PIM a connection.  
+In this step, we are going to associate this connection to the connector you've built (or downloaded from the marketplace).
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-3.md]:
```suggestion
In this step now, we are going to associate this connection to the connector you've built (or downloaded from the marketplace).
```

File: content/getting-started/connect-the-pim-4x/step-3.md [hunk 23]
```diff
@@ -0,0 +1,28 @@
...
+
+By doing so, you will be able to easily monitor the data flows that are exchanged between your connector and the PIM.
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-3.md]:
```suggestion
By doing so, you will be able to easily monitor the data flows between your connector and the PIM.
```

File: content/getting-started/connect-the-pim-4x/step-3.md [hunk 24]
```diff
@@ -0,0 +1,28 @@
...
+3. In the `Credentials` section of the screen, you will find the "key" to the binding between your connection and your connectors: the credentials.
+![Connection credentials](/img/getting-started/connection-credentials.png)
+4. Thanks to the _copy_ icon on the left side of the table, copy each credential field: the client id, the secret, the username and password.
+![Copy icon](/img/getting-started/connection-credentials-copy-icon.png)
+5. Paste them into your connector so that your connector will use these credentials to be authentified over the API.
+
+That's it! :tada:
+
+Whenever your connector will start making API calls, you will be able to check whether your connector is actually working. Indeed, we got you covered with the **Connection dashboard**.  
+To access this screen, click on the `Activity` menu and then on the `Connection dashboard` submenu.
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-3.md]: Should we recall this dashboard is available for 4.x and beyond, only?

File: content/getting-started/connect-the-pim-4x/step-3.md [hunk 25]
```diff
@@ -0,0 +1,28 @@
...
+
+You can follow here the number of products that were updated or created during the last seven days. In our tutorial, it's perfect, because we are updating/creating products coming from our ERP. So we can verify that products are actually created/updated by our connector. 😊
+
+::: info
+In 4.0, you can only follow the creation and update of products through the Connection dashboard. But don't worry, in next versions, you will be able to also follow the distribution of products sent to your various destinations, in the case you chose `Destination`as your connection flow type. So stay tuned!
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/step-3.md]:
```suggestion
In 4.0, you can only follow the creation and update of products through the Connection dashboard. But don't worry, in next versions, you will also be able to follow the distribution of products sent to your various destinations, in the case you chose `Destination`as your connection flow type. Stay tuned!
```

File: content/getting-started/connect-the-pim-4x/welcome.md [hunk 26]
```diff
@@ -0,0 +1,33 @@
...
+
+::: tips
+Connections will be achieved with the help of the API, as the API is the best way to connect the PIM. Wondering why? Discover the reasons [here](/documentation/introduction.html#why-should-you-use-our-api)!
+:::
+
+## Prerequisites
+
+<i class="fa fa-check-square"></i> **You have access to a 4.x Akeneo PIM.**  
+This can seem pretty obvious. But yeah! You will need a PIM if you want to connect it to any third parties.  
+And of course, you also have to make sure that you can access it from where you are requesting.
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-4x/welcome.md]: I would add something like:
If you do not have access to a 4.x version of Akeneo PIM, please get in touch with Akeneo team, or your Channel Manager, etc.

File: content/getting-started/connect-the-pim-old/step-2.md [hunk 27]
```diff
@@ -0,0 +1,125 @@
+# _Step 2_ | Generate your credentials
+
+In this step, you are going to generate your credentials. :rocket:
+
+Indeed, you will need an API user, a password, a client ID and a secret if you want the connector you just cooked, to work. So let's get those credentials!
+
+::: info
+_Best practice_  
+We strongly advise you to create **one set of credentials** for each one of your connectors.  
+Indeed, it will ease a lot your migration to the PIM v4, in case you are planning it.
```
Comment (cbarbe) [inline content/getting-started/connect-the-pim-old/step-2.md]:
```suggestion
Indeed, it will ease a lot your migration to the PIM v4, in case you are planning for it.
```

File: content/getting-started/connect-the-pim-4x/step-3.md (hunk 24 above)
Comment (NolwennP) [reply content/getting-started/connect-the-pim-4x/step-3.md]: This tutorial is dedicated to v4 version only. Connection also are only available starting from the 4.0, so...

File: content/getting-started/connect-the-pim-4x/welcome.md (hunk 26 above)
Comment (NolwennP) [reply content/getting-started/connect-the-pim-4x/welcome.md]: You're thinking about which kind of persona here?

File: content/getting-started/connect-the-pim-old/step-1.md [hunk 28]
```diff
@@ -0,0 +1,120 @@
...
+
+### 5- Think about your architecture
+Choose the architecture you want for your connector. Indeed, you will have two options here:
+    - You have access to the third-party code source and can develop your connector directly in it.
+    - You can otherwise create a middleware that will sit right in the middle of the tird-party you want to connect and the PIM.
+
+#### In this tutorial
+_For our example, we choose the second option there is little chance that your ERP allows developing into its base code. So you will have to develop your connector as a middleware._
+
+### 6- Ochestrate your global connector
```
Comment (NolwennP) [inline content/getting-started/connect-the-pim-old/step-1.md]:
```suggestion
### 6- Orchestrate your global connector
```

File: content/getting-started/connect-the-pim-old/step-1.md [hunk 29]
```diff
@@ -0,0 +1,120 @@
...
+
+### Think scalability and performance
+
+It is really important that you take into consideration the scalability and performance of your connector.
+
+As stated previously, again, we advise you to start small. Benchmark your connector with a small volume, then test it with a real-life catalog volume.
+
+To help you in this task, here are examples of questions you can ask yourself:
+- _"How many products, assets, reference entities or options your connector will handle?"_
+- _"How big are your products, assets or reference entities in termes of number of attributes completed?"_
```
Comment (NolwennP) [inline content/getting-started/connect-the-pim-old/step-1.md]:
```suggestion
- _"How big are your products, assets or reference entities in terms of number of attributes completed?"_
```

File: content/getting-started/connect-the-pim-old/step-1.md [hunk 30]
```diff
@@ -0,0 +1,120 @@
...
+#### In this tutorial
+_For our example, you will need to create and update products in the PIM so we will mainly need this endpoint: [Patch](/api-reference.html#patch_products) on products._
+
+### 5- Think about your architecture
+Choose the architecture you want for your connector. Indeed, you will have two options here:
+    - You have access to the third-party code source and can develop your connector directly in it.
+    - You can otherwise create a middleware that will sit right in the middle of the tird-party you want to connect and the PIM.
+
+#### In this tutorial
+_For our example, we choose the second option there is little chance that your ERP allows developing into its base code. So you will have to develop your connector as a middleware._
```
Comment (NolwennP) [inline content/getting-started/connect-the-pim-old/step-1.md]:
```suggestion
_For our example, we choose the second option as little chances are that your ERP accepts external development into its base code. Here, you will have to develop your connector as a middleware._
```

File: content/getting-started/connect-the-pim-old/step-1.md [hunk 31]
```diff
@@ -0,0 +1,120 @@
...
+6. Ochestrate your global connector
+
+Those steps are detailed below, illustrated by our tutotrial example.
+
+### 1- Have your connection use case in mind
+
+We know it can seem a bit obvious but it really is the first step for a successful connection between the PIM and your third-party. You need to have at least the global scenario of your connection in mind.
+
+#### In this tutorial
+_You want to connect your ERP. Each day, new products are created into your ERP. You want those new products to be sent over to the PIM so that you can enrich them, in the PIM, the day after._
```
Comment (NolwennP) [inline content/getting-started/connect-the-pim-old/step-1.md]:
```suggestion
_You want to connect your ERP. Each day, new products are created in your ERP. You want those new products to be sent over to the PIM so that you can enrich them, in the PIM, the day after._
```

File: content/getting-started/connect-the-pim-old/step-1.md [hunk 32]
```diff
@@ -0,0 +1,120 @@
...
+
+Also, don't forget to think about the fact that every API call you are doing has an impact on the **PIM load**. Always keep in mind that your connector won't be the only one connected to the PIM. Try to have a load print as low as possible. To do so, you can use caches for example.
+
+### Test, test and test
+
+Testing should not be optional. You will need to test your full use case and also focus on all the edge cases that can happen.
+
+### Use the PHP client
+
+If you want to use PHP to build your connector, we highly recommend the use of our PHP client.
```
Comment (NolwennP) [inline content/getting-started/connect-the-pim-old/step-1.md]:
```suggestion
If you want to use PHP to build your connector, we highly recommend the use of [our PHP client](/php-client/introduction.html).
```

File: content/getting-started/connect-the-pim-4x/step-2.md (hunk 19 above)
Comment (NolwennP) [reply content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
If you want to use PHP to build your connector, we highly recommend the use of [our PHP client](/php-client/introduction.html).
```

File: content/getting-started/connect-the-pim-4x/step-2.md (hunk 16 above)
Comment (NolwennP) [reply content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
- _"How many [products](/documentation/resources.html#product), [product models](/documentation/resources.html#product-model-2x-only), [assets](/documentation/resources.html#asset-since-21-and-ee-only-deprecated), [reference entity records](/documentation/resources.html#reference-entity-record-3x-and-ee-only) or [options](/documentation/resources.html#attribute-option) your connector will handle?"_
```

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 33]
```diff
@@ -0,0 +1,120 @@
...
+
+### Think scalability and performance
+
+It is really important that you take into consideration the scalability and performance of your connector.
+
+As stated previously, again, we advise you to start small. Benchmark your connector with a small volume, then test it with a real-life catalog volume.
+
+To help you in this task, here are examples of questions you can ask yourself:
+- _"How many products, assets, reference entities or options your connector will handle?"_
+- _"How big are your products, assets or reference entities in terms of number of attributes completed?"_
```
Comment (NolwennP) [inline content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
- _"How big are your products, product models, assets or reference entities in terms of number of attributes completed?"_
```

File: content/getting-started/connect-the-pim-old/step-1.md [hunk 34]
```diff
@@ -0,0 +1,120 @@
...
+Another option is to provide a configuration file to input this configuration. It's usually faster to develop but keep in mind that you will be far from delivering the best UX for your final users. :wink:
+
+### Think scalability and performance
+
+It is really important that you take into consideration the scalability and performance of your connector.
+
+As stated previously, again, we advise you to start small. Benchmark your connector with a small volume, then test it with a real-life catalog volume.
+
+To help you in this task, here are examples of questions you can ask yourself:
+- _"How many products, assets, reference entities or options your connector will handle?"_
```
Comment (NolwennP) [inline content/getting-started/connect-the-pim-old/step-1.md]:
```suggestion
- _"How many [products](/documentation/resources.html#product), [product models](/documentation/resources.html#product-model-2x-only), [assets](/documentation/resources.html#asset-since-21-and-ee-only-deprecated), [reference entity records](/documentation/resources.html#reference-entity-record-3x-and-ee-only) or [options](/documentation/resources.html#attribute-option) your connector will handle?"_
```

File: content/getting-started/connect-the-pim-old/step-1.md [hunk 35]
```diff
@@ -0,0 +1,120 @@
...
+
+### Think scalability and performance
+
+It is really important that you take into consideration the scalability and performance of your connector.
+
+As stated previously, again, we advise you to start small. Benchmark your connector with a small volume, then test it with a real-life catalog volume.
+
+To help you in this task, here are examples of questions you can ask yourself:
+- _"How many [products](/documentation/resources.html#product), [product models](/documentation/resources.html#product-model-2x-only), [assets](/documentation/resources.html#asset-since-21-and-ee-only-deprecated), [reference entity records](/documentation/resources.html#reference-entity-record-3x-and-ee-only) or [options](/documentation/resources.html#attribute-option) your connector will handle?"_
+- _"How big are your products, assets or reference entities in terms of number of attributes completed?"_
```
Comment (NolwennP) [inline content/getting-started/connect-the-pim-old/step-1.md]:
```suggestion
- _"How big are your products, product models, assets or reference entities in terms of number of attributes completed?"_
```

File: content/getting-started/connect-the-pim-4x/step-2.md (hunk 1 above)
Comment (camillebourjade) [reply content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
It is the case in our example in this tutorial. We want to connect the PIM to our ERP but the marketplace doesn't provide any connector that covers this use case.  
```

File: content/getting-started/connect-the-pim-4x/step-2.md (hunk 2 above)
Comment (camillebourjade) [reply content/getting-started/connect-the-pim-4x/step-2.md]:
```suggestion
In the event you can't find an existing connector on our marketplace, you will have to develop it. 
```

File: content/getting-started/connect-the-pim-4x/welcome.md (hunk 26 above)
Comment (cbarbe) [reply content/getting-started/connect-the-pim-4x/welcome.md]: As a System Integrator or a Technology Provider, I want to develop a connector but do not have access to the latest Akeneo version.
-> OK then ask your Channel Manager (I'm considered as one in the Portal) or get in touch with partners@akeneo.com.
Thanks :)

File: content/getting-started/connect-the-pim-old/step-2.md [hunk 36]
```diff
@@ -0,0 +1,112 @@
...
+:::
+
+Depending on your PIM version (v1.7 or v2/v3), there are two methods to generate your client ID and secret.
+
+If you are using a v2 or V3 PIM, [create an API connection](#by-creating-an-API-connection).  
+If you are using a v1.7, [launch a command line](#by-command-line-for-v17-pim-only).
+
+### By creating an API connection
+
+We will create an API connection. We will call it, for the purpose of this tutorial: `ERP`
```
Comment (camillebourjade) [inline content/getting-started/connect-the-pim-old/step-2.md]:
```suggestion
We will create an API connection. We will call it `ERP` for the purpose of this tutorial.
```

File: content/getting-started/connect-the-pim-old/step-2.md [hunk 37]
```diff
@@ -0,0 +1,112 @@
...
+
+::: warning
+This feature is only available since the 2.0 version. If you are trying to use the API on a 1.7 PIM, please have a look to the [second method below](#get-your-credentials-via-command-line-).
```
Comment (camillebourjade) [inline content/getting-started/connect-the-pim-old/step-2.md]:
```suggestion
This feature is only available since the 2.0 version. If you are trying to use the API on a 1.7 PIM, please take a look at the [second method below](#get-your-credentials-via-command-line-).
```

File: content/getting-started/connect-the-pim-old/step-2.md [hunk 38]
```diff
@@ -0,0 +1,112 @@
...
+:::
+
+1. Click on the `System` menu.
+1. Click on the `API connections` submenu.
+1. Click on the `Create` button.
+![API connections empty screen](/img/getting-started/api-connections-empty-screen.png)
+The following popin should be then displayed.
+![API connection creation popin](/img/getting-started/api-connection-creation-popin.png)
+1. Enter `ERP` in the label field and click on the `Save` button.  
+You API connection appears in the API connections table and should look like this:
```
Comment (camillebourjade) [inline content/getting-started/connect-the-pim-old/step-2.md]:
```suggestion
Your API connection appears in the API connection table and should look like this:
```

File: content/getting-started/connect-the-pim-old/step-2.md [hunk 39]
```diff
@@ -0,0 +1,112 @@
...
+![My very first API connection](/img/getting-started/my-erp-connection.png)
+
+And that's it! :tada:
+
+::: panel-link Now that you generated your credentials, let's move on to [The connector configuration](/getting-started/connect-the-pim-old/step-3.html)
+:::
+
+### By command line for v1.7 PIM only
```
Comment (camillebourjade) [inline content/getting-started/connect-the-pim-old/step-2.md]:
```suggestion
### Use a command line for the v1.7 PIM only
```

File: content/getting-started/connect-the-pim-old/step-3.md [hunk 40]
```diff
@@ -0,0 +1,18 @@
+# Configure your connector
+
+Thanks to step 1, you now have a wonderful connector.
+Thanks to the step 2, you now have:
```
Comment (camillebourjade) [inline content/getting-started/connect-the-pim-old/step-3.md]:
```suggestion
Thanks to step 2, you now have:
```

File: content/getting-started/connect-the-pim-old/step-3.md [hunk 41]
```diff
@@ -0,0 +1,18 @@
+# Configure your connector
+
+Thanks to step 1, you now have a wonderful connector.
```
Comment (NolwennP) [inline content/getting-started/connect-the-pim-old/step-3.md]:
```suggestion
Thanks to step 1, you now have a wonderful connector.  
```

File: content/getting-started/connect-the-pim-old/welcome.md [hunk 42]
```diff
@@ -0,0 +1,33 @@
+# 👋 Welcome in this guide!
+
+This tutorial will teach you:
+> How to connect the PIM with the API on a v1.7, v2 or v3 PIM
+
+## Goal
+At the end of this tutorial, you will know what are the primary steps to properly connect the PIM.
```
Comment (camillebourjade) [inline content/getting-started/connect-the-pim-old/welcome.md]:
```suggestion
At the end of this tutorial, you will know what the primary steps to properly connect the PIM are.
```

File: content/getting-started/connect-the-pim-old/welcome.md [hunk 43]
```diff
@@ -0,0 +1,33 @@
...
+This tutorial is dedicated to version of the PIM older than the v4. If you are working on a v4 PIM, we also got you covered! :wink: There is a tutorial for you right [here](/getting-started/connect-the-pim-4x/welcome.html).
+::: 
+
+<i class="fa fa-check-square"></i> **You already know to which third party you want to connect your PIM.**  
+It can be one of the following: an ERP, a DAM, an ecommerce platform, a syndication platform, a translation tool, a publishing tool... You name it! Pick one for which you need a connection with your PIM.
+
+As an example, in this tutorial, we will consider that we want to connect the PIM to your ERP.
+
+You will also have to know the use case for your connection.  
+In this tutorial, for our ERP example, let's say each day, new products are referenced into your ERP. You want those new products to be sent over to your PIM so that you can enrich them, in the PIM, the day after.
```
Comment (camillebourjade) [inline content/getting-started/connect-the-pim-old/welcome.md]:
```suggestion
In this tutorial, for our ERP example, let's say everyday day, new products are referenced into your ERP. You want those new products to be sent over to your PIM so that you can enrich them, in the PIM, the day after.
```

Comment (cbarbe) [reply content/getting-started/connect-the-pim-old/welcome.md]: everyday day

Comment (NolwennP) [reply content/getting-started/connect-the-pim-old/welcome.md]:
```suggestion
In this tutorial, for our ERP example, let's say every day, new products are referenced into your ERP. You want those new products to be sent over to your PIM so that you can enrich them, in the PIM, the day after.
```

File: content/getting-started/connect-the-pim-4x/welcome.md (hunk 26 above)
Comment (NolwennP) [reply content/getting-started/connect-the-pim-4x/welcome.md]:
```suggestion
And of course, you also have to make sure that you can access it from where you are requesting.

::: tips
If you do not have access to a 4.x version of Akeneo PIM, please get in touch with [Akeneo team](mailto:partners@akeneo.com) or your Channel Manager. 
:::
```

File: content/getting-started/connect-the-pim-4x/step-1.md [hunk 44]
```diff
@@ -0,0 +1,45 @@
...
+Well, maybe you are not familiar with what a connection is. Let me help you here.
+
+In a few words, a connection can be considered as a door for the PIM. It allows you to push data into or pull data from the PIM.
+
+Connections are directly declared into the UI of the PIM. To declare a connection, you should follow this simple rule: **One connection corresponds to one specific data flow**.
+
+Alright!
+
+::: info
+Want to dig a bit deeper into the connection notion? Visit the dedicated help center [article about it](https://help.akeneo.com/articles/what-is-a-connection.html). :wink:
```
[the remark this answers is not in the document]
Comment (NolwennP) [reply content/getting-started/connect-the-pim-4x/step-1.md]: It's normal, the doc is not deployed yet. Everything will be deployed for the release.

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 45]
```diff
@@ -0,0 +1,120 @@
...
+
+## Find your connector on our marketplace
+
+We have good news for you. Maybe the connector you have in mind already exists!
+
+Indeed, we have a marketplace on which our dear ecosystem publishes its own tools and connectors.
+
+So, we definitely recommend you to take a look at it and try to find a connector that can suit your use case. It will save you a lot of time!
+
+1. Navigate to the [marketplace](https://marketplace.akeneo.com/extensions/add-ons?edition=all&version=all&api_use=1&sort=date) website.
```
[the remark this answers is not in the document]
Comment (NolwennP) [reply content/getting-started/connect-the-pim-4x/step-2.md]: Ah, I'll change that.

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 46]
```diff
@@ -0,0 +1,120 @@
...
+
+#### In this tutorial
+_You want to connect your ERP. Each day, new products are created in your ERP. You want those new products to be sent over to the PIM so that you can enrich them, in the PIM, the day after._
+
+### 2- Find out what are the third-party connection possibilities
+
+Take a closer look at the software you want to connect to. How can you extract the data you will need? Does it have an API? Does it allow exporting flat files?  
+
+#### In this tutorial
+_In our example, let's say that you took a look at your ERP and discovered that, like many ERPs, it is a bit old-school. It only provides exports of products in XML files._
```
[the remark this answers is not in the document]
Comment (NolwennP) [reply content/getting-started/connect-the-pim-4x/step-2.md]: I say "like many", maybe not theirs. ;)

File: content/getting-started/connect-the-pim-4x/step-2.md [hunk 47]
```diff
@@ -0,0 +1,120 @@
...
+
+### 4- Explore our API reference
+In this step, you will need to find in our [API reference](api-reference-index.html) the endpoints you will need to achieve such connection. 
+
+#### In this tutorial
+_For our example, you will need to create and update products in the PIM so we will mainly need this endpoint: [Patch](/api-reference.html#patch_products) on products._
+
+### 5- Think about your architecture
+Choose the architecture you want for your connector. Indeed, you will have two options here:
+    - You have access to the third-party code source and can develop your connector directly in it.
```
[the remark this answers is not in the document]
Comment (NolwennP) [reply content/getting-started/connect-the-pim-4x/step-2.md]: Ok, I'll add a warning about that.
